adserver/templates/adserver/advertiser/advertisement-bulk-create-template.csv

```diff
@@ -0,0 +1,2 @@
+Name,Live,Link URL,Image URL,Headline,Content,Call to Action
+Your Ad Name (required),"Whether to make your ad immediately live (required, true or false)",Your ad landing page URL (required),"Your ad image URL (required, the size must be 240*180px)",Ad headline (optional),"Ad content(required, the length of the headline, content and CTA combined must be 100 characters or less)",Ad CTA (optional)
```

Comment:
Is there a way to specify 2 header fields on import? I'm wondering if we want to let folks keep the instructions here even after they've created the ads for import? Seems nice for "documenting" the ads on their side in the future, but not a huge requirement.

Reply:
I'm not sure exactly what you mean. As in, set on an ad that this was bulk-imported?

Reply:
Just it would be nice for this documentation to stay inside the CSV when the users are working with it. Not a huge deal, but would be nice.

Reply:
I've seen this pattern on other products I've used. We could check if the 2nd line matches this file's 2nd line exactly and ignore it in that case but we want to make sure we don't ignore a row the client uses.

Reply:
Yea, not something we need for this version necessarily, perhaps we can put this data somewhere else in the docs/UI as well?

Reply:
The UI is probably the best place of those two. Maybe some sort of collapsable element where it can go?